tests/errors/errors.php

```diff
@@ -46,3 +46,16 @@
 } finally {
     // Comment
 }
+
+// PHP 8.0 non-capturing exception catch
+try {
+    // Something
+} catch (\Exception) {
+    // Nothing
+}
```

Comment:
Is it worth making the example something that will actually be reformatted?

Reply:
I think in this case it's not needed; we're testing the formatting of try/catch already thoroughly, here we just make sure that non-capturing exceptions also work.

Reply:
Makes sense, I think we can merge this then.